core/src/main/java/org/elasticsearch/index/query/TemplateQueryBuilder.java

```diff
@@ -35,14 +42,8 @@
 
     /** Template to fill. */
     private Template template;
-    /** Parameters to fill the template with. */
-    private Map<String, Object> vars;
-    /** Template to fill.*/
-    private String templateString;
 
-    private ScriptService.ScriptType templateType;
```

Comment:
+1 on removing these as essentially they are encapsulated in the Template above.